narwhal/config/tests/snapshots/config_tests__parameters_import.snap

```diff
@@ -1,9 +1,10 @@
 ---
-source: config/tests/config_tests.rs
+source: narwhal/config/tests/config_tests.rs
 expression: params
 ---
 {
-  "header_size": 1000,
+  "header_num_of_batches_threshold": 32,
+  "max_header_num_of_batches": 100,
```

Comment:
The insta-generated default here for `max_hader_num_of_batches` is 100, whereas it's 1000 above. What gives?

Reply:
This actually comes from the `config_tests` - where I have the parameter `100` (as a differentiator), that's why the snapshot is 100